### PR TITLE
Units

### PR DESCRIPTION
**Pull request by selassie-eye**
Implemented Unit interfaces, added attributes

### CODE REVIEW

File: src/main/game/entities/units/UnitNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package game.entities.units;
+
+/**
+ * Created by khariollivierre on 2/25/17.
+ */
+public class UnitNotFoundException extends Throwable {
```
Comment (gdscheele) [inline src/main/game/entities/units/UnitNotFoundException.java]: this should extend Exception not Throwable. Throwable is the super class for both Exception and Error

Comment (gdscheele) [inline src/main/game/entities/units/UnitNotFoundException.java]: this should extend Exception not Throwable. Throwable is the super class for both Exception and Error

File: src/main/game/entities/units/UnitStats.java [hunk 2]
```diff
@@ -9,40 +9,44 @@
...
+                    this.defPow = 7;
+                    this.armor = 3;
+                    this.speed = 2;
+                    this.health = 10;
+                    this.upkeep = .1f;
+                    break;
+                default:
+                    throw new UnitNotFoundException();
+            }
+        } catch(UnitNotFoundException e){}
```
Comment (gdscheele) [inline src/main/game/entities/units/UnitStats.java]: should not catch this exception here. The caller should handle it.

**Comment by karthshen** [comment]
Gavin's request has already been met, I'll dismiss the review and merge this branch.

**Pull request merged**